elements/rh-tag/rh-tag.ts

```diff
@@ -90,11 +77,19 @@ export class RhTag extends LitElement {
               teal: color === ('cyan' as 'blue' /* cyan deprecated */) || color === 'teal',
               [variant]: true,
               [color]: true })}">
+        <!--
+          slot:
+            summary: Contains the labels's icon, e.g. web-icon-alert-success.
```

Comment:
We have two columns for the slot: summary and description in the docs accordion. They seem to be inconsistently used. If you don't have multiple attributes, such as a slot name and part, and use the YAML format, it appears that the comment text is placed in the description field. This may be by design; if so, ignore.